homeassistant/components/shelly/switch.py

```diff
@@ -60,6 +65,12 @@ class BlockSwitchDescription(BlockEntityDescription, SwitchEntityDescription):
     entity_registry_enabled_default=False,
 )
 
+MOTION_SWITCH = BlockSwitchDescription(
+    key="sensor|motionActive",
+    name="Motion active",
```

Comment:
I'm not sure if `Motion active` is a clear term for the end user. Maybe `Motion detection` would be better. What do you think?

Reply:
Done